pkg/security/security_profile/dump/activity_dump.go

```diff
@@ -308,9 +308,11 @@ func (ad *ActivityDump) SetTimeout(timeout time.Duration) {
 func (ad *ActivityDump) updateTracedPid(pid uint32) {
 	// start by looking up any existing entry
 	var cookie uint32
-	_ = ad.adm.tracedPIDsMap.Lookup(pid, &cookie)
-	if cookie != ad.LoadConfigCookie {
-		_ = ad.adm.tracedPIDsMap.Put(pid, ad.LoadConfigCookie)
+	if ad.adm != nil { // it could be nil when running unit tests
```

Comment:
could we create an adm in the UT

Reply:
Not easily, and to make it works it should also have the tracePIDsMap initialized and IMHO I think that defeat the purpose of having simple UT with limited scope